oss-training-2019-2.md

```diff
@@ -0,0 +1,181 @@
+# 第２回勉強会 Githubハンズオン
+
+
+---
+
+## 目次
+
+- アカウント作成
+- 公開鍵の登録
+- リポジトリ作成
+- コード公開
+- Issue作成
+- PR作成
+
+---
+
+# アカウント作成
+
+---
+
+## アカウント作成①
+
+https://github.com/join にアクセスする。
+!(img/1-01.PNG)
```

Comment:
add `[]` between `!` and `(` like follow.
`![](img/1-01.PNG)`

Reply:
done, thanks